apps/web/next.config.js

```diff
@@ -1,4 +1,8 @@
 module.exports = {
 	reactStrictMode: true,
-	transpilePackages: ["ui"]
+	transpilePackages: ["ui"],
+	eslint: {
+		ignoreDuringBuilds: true
```

Comment:
![image](https://github.com/stamford-syntax-club/course-compose/assets/24469918/0a0e8207-51bd-4144-9e71-d23247ead214)

Reply:
FYI, there are SO many linter errors in this PR it's not even funny.
I have already disabled many rules, but there are still some that I'd like to maintain, even though it's an afterthought at this point.

Reply:
had to open with VSCode to see. Fixed a bunch of them -*-